ultils/index.js

```diff
@@ -6,7 +6,7 @@ export function getCurrentDate() {
   const date = new Date();
   const year = date.getFullYear();
   const month = date.getMonth() + 1;
-  const dates = date.getDate();
+  const dates = date.getDate() < 10 ? "0" + date.getDate() : date.getDate();
```

Comment:
You want a leading zero here? then you can use padStart: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart